tfe/resource_tfe_workspace.go

```diff
@@ -90,6 +90,18 @@ func resourceTFEWorkspace() *schema.Resource {
 				Default:  true,
 			},
 
+			"global_remote_state": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
Because this isn't a true default and configurable for Terraform Enterprise, I think `Computed` should be set here instead of a `Default`.

[Computed](https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#computed) is a schema behavior that in this case means that the provider cannot predict the value at plan-time and the value will be provided by the platform (Which will be TFC _or_ TFE - it could default to `false`!) in the absence of it being specified by the user in the configuration.

Reply:
Ah, okay, makes sense!